docs/index.rst

```diff
@@ -26,28 +26,30 @@
   <div class="grid-x grid-margin-x hs">
 
 .. topic-box::
-  :title: Let us manage your DB
+  :title: ScyllaDB Cloud
   :link: https://cloud.docs.scylladb.com
   :class: large-4
-  :anchor: Get Started with Scylla Cloud
+  :anchor: ScyllaDB Cloud Documentation
 
-  Take advantage of Scylla Cloud, a fully-managed database-as-a-service.
+  Simplify application development with the fully managed DBaaS.
 
 .. topic-box::
-  :title: Manage your own DB
+  :title: ScyllaDB On-Premise
```

Comment:
Not sure about this one
Most of ScyllaDB deployments are on clouds like AWS< GCP or Azure, not on On-Premise.